cargo-near/src/commands/build_command/mod.rs

```diff
@@ -1,30 +1,18 @@
 use std::ops::Deref;
-use std::process::{id, Command};
-use std::time::{SystemTime, UNIX_EPOCH};
 
-#[cfg(unix)]
-use nix::unistd::{getgid, getuid};
+use crate::{types::manifest::CargoManifestPath, util};
 
-use color_eyre::{
-    eyre::{ContextCompat, WrapErr},
-    owo_colors::OwoColorize,
-};
-use serde::Deserialize;
-
-use crate::{
-    types::{manifest::CargoManifestPath, metadata::CrateMetadata},
-    util,
-};
-
-pub mod build;
+mod build;
+mod docker;
+pub const INSIDE_DOCKER_ENV_KEY: &str = "NEAR_BUILD_ENVIRONMENT_REF";
```

Comment:
Given that the latest naming according to the semi-approved [NEP-330 extension is `build_info.build_environment`](https://github.com/near/NEPs/pull/533/files#diff-4955342da1cab11a6777c89e1ae874f185c844101af84775c413b87098aa9a12R104) and we use [`CARGO_*` variables in `near-sdk-rs`](https://github.com/near/near-sdk-rs/blob/master/near-sdk-macros/src/core_impl/contract_metadata/mod.rs#L51-L58), let's name it `CARGO_NEAR_BUILD_ENVIRONMENT`.

We will also need to pass `CARGO_NEAR_SOURCE_CODE_SNAPSHOT`, `CARGO_NEAR_CONTRACT_PATH`, and `CARGO_NEAR_BUILD_COMMAND`, so near-sdk-rs can pick those up and include in the metadata.